utility/script_config_dict.py

```diff
@@ -180,6 +180,12 @@ class ScriptConfigInfo(BaseScriptConfigInfo, total=False):
                 'type': str,
                 'default': '/var/yang/all_modules',
             },
+            {
+                'flag': '--official-source',
+                'help': 'If this is an official souce of an SDOs modules, set this option to the name of the SDO',
+                'type': str,
+                'default': 'None',
```

Comment:
'None' as a string can be confusing, maybe just use  ```'default': '',```?

Reply:
Yep, that was unintentional.